### PR TITLE
Add search bar to scorecard page on mobile 

### PR DESCRIPTION
**Pull request by justinebreuch**
## Description

Addresses: [Add search to scorecard page](https://tables.area120.google.com/u/1/workspace/bxQE2CPzwu_b4sJEkotkUM/table/bX_kMNBW8LA35n6l5Hz_co/view/auSM-s1cO9l3pAsYHmOk1E)

Add search to scorecard on mobile 

## Testing and Reviewing

Desktop: 
image

Mobile:
image

### CODE REVIEW

File: client/src/components/scorecard_page/ScorecardMapViewBar.vue [hunk 1]
```diff
@@ -115,8 +117,13 @@ export default defineComponent({
   background-color: $warm-grey-100;
 }
 
+@media screen and (max-width: 470px) {
+  .container {
+    justify-content: center;
```
Comment (justinebreuch) [inline client/src/components/scorecard_page/ScorecardMapViewBar.vue:122]: No way to do this in Bulma

Comment (beekley) [reply client/src/components/scorecard_page/ScorecardMapViewBar.vue:122]: does https://bulma.io/documentation/helpers/typography-helpers/#alignment help?

File: client/src/views/ScorecardView.vue [hunk 2]
```diff
@@ -12,14 +12,14 @@
       <PredictionPanel />
 
       <!-- Only display the side-panel, full-width on mobile. -->
-      <ScorecardMapViewBar class='is-hidden-mobile' />
-      <div class='columns is-variable is-centered is-gapless'>
+      <ScorecardMapViewBar />
```
Comment (beekley) [inline client/src/views/ScorecardView.vue:15]: How does this look on mobile with pills?

**Review by beekley — COMMENTED** [review]
LGTM, but one quick question.

File: client/src/views/ScorecardView.vue (hunk 2 above)
Comment (justinebreuch) [reply client/src/views/ScorecardView.vue:15]: The pills appear on desktop and disappear on mobile 
image

**Pull request merged**